codalab/model/bundle_model.py

```diff
@@ -1001,12 +1001,16 @@ def transition_bundle_running(self, bundle, worker_run, row, user_id, worker_id,
             'remote': worker_run.remote,
             'cpu_usage': cpu_usage,
             'memory_usage': memory_usage,
+            'data_size': worker_run.disk_utilization,
```

Comment:
FYI, with regard to what we spoke about in the previous meeting, due to this change the size of the run bundle _is_ being updated on each bundle checkin, and so the updates to the user disk quota (which are made below by the increment_user_disk_used function) won't be overwritten when update_disk_metadata is called.